.github/workflows/actions.yml

```
@@ -330,9 +330,9 @@ jobs:
           name: binaries-${{ github.job }}
           path: pfx/**/*
 
-  # Only verify build support on old macOS
-  macos-11:
-    runs-on: macos-11
+  # Only verify build support on older macOS and SDK
+  macos-12:
+    runs-on: macos-12
     needs: [ what-to-make ]
     if: false  # ${{ needs.what-to-make.outputs.make-mac == 'true' }}
```

You forgot to revert that line
```suggestion
    if: ${{ needs.what-to-make.outputs.make-mac == 'true' }}
```

---

Thanks!